.gitignore

```diff
@@ -17,3 +17,4 @@ docs/source/generated/
 docs/build/
 cache/
 .tox/
+tools/
```

Comment:
accident?

Reply:
confusion/accident :(   Ok I removed that.